make_prg/subcommands/update.py

```diff
@@ -56,6 +56,17 @@ def check_if_is_update_file(argument: str):
             "Filepath containing denovo sequences. Should point to a denovo_paths.txt file"
         ),
     )
+    subparser_update_prg.add_argument(
+        "--long-deletion-threshold",
+        dest="long_deletion_threshold",
+        action="store",
+        type=int,
+        default=10,
```

Comment:
This seems a pretty low default? Should we maybe use something a bit bigger?

Reply:
I had issues with deletions longer than 10, although I don't really remember which values I used, and we ignore few deletions by using 10 as the threshold in paper data. [I increased the log level when we ignore a variant due to a `TooLongDeletion` error](https://github.com/iqbal-lab-org/make_prg/pull/37/commits/699a95993a329c16199d0dbcafabf0cc12701c82), so that the user can easily count how many variants is lost due to this. In the error message, the variant's [delta length is also reported](https://github.com/leoisl/make_prg/blob/699a95993a329c16199d0dbcafabf0cc12701c82/make_prg/update/denovo_variants.py#L56), so a better tuning of this parameter can be done. But I should note it was a single test I did to set this variable default, so definitely 10 is not the ideal value (it is a bit complicated, as the ideal value I think is dataset- and performance-dependent). Anyway, I did not invest much time worrying about this as this is just a temporary solution - once these `make_prg  update` PR series are merged, then I will start working on the next `make_prg` version, which the only objective is to properly deal with indels/long variants (instead of the temporary solution we have here, which breaks these variants into several smaller ones).

Given all this info, please tell me what is your decision: 1) if this is fine as default as is just a temporary solution; or 2) we should improve this value to a better one (which one?) as the version that properly deals with indels/long variants can take some time to be implemented.

Reply:
Ok, fine to leave as is